[Brandhub] Make tafl component adapted to different sized boards

MartinREMY42 opened the pull request.

acieroid committed:
Add package-lock

acieroid committed:
Add package-lock

acieroid committed:
[deployment] Try to get deployment working

acieroid committed:
[deployment] Try to get deployment working

acieroid committed:
[deployment] Use secret variables for firebaseConfig

acieroid committed:
[deployment] Use secret variables for firebaseConfig

acieroid committed:
[deployment] Fix typo in script

acieroid committed:
[deployment] Fix typo in script

acieroid committed:
[deployment] Different approach for deployment script

acieroid committed:
[deployment] Different approach for deployment script

acieroid committed:
[deployment] Fix more typos in deployment script

acieroid committed:
[deployment] Fix more typos in deployment script

acieroid committed:
[deployment] One more typo

acieroid committed:
[deployment] One more typo

acieroid committed:
[deployment] Improve netlify production build

acieroid committed:
[deployment] Improve netlify production build

acieroid committed:
[deployment] Fix typo in package.json

acieroid committed:
[deployment] Fix typo in package.json

acieroid committed:
[deployment] Try to get chromedriver working

acieroid committed:
[deployment] Try to get chromedriver working

acieroid committed:
[deployment] Update dependencies

acieroid committed:
[deployment] Update dependencies

acieroid committed:
[deployment] Use webdriver-manager to get chrome to work hopefully

acieroid committed:
[deployment] Use webdriver-manager to get chrome to work hopefully

acieroid committed:
[deployment] Force the use of python 3

acieroid committed:
[deployment] Force the use of python 3

acieroid committed:
[deployment] Fix pip call

acieroid committed:
[deployment] Fix pip call

acieroid committed:
[deployment] Update screenshotter

acieroid committed:
[deployment] Update screenshotter

acieroid committed:
[deployment] Add debugging code

acieroid committed:
[deployment] Add debugging code

acieroid committed:
[deployment] Try to set CHROME_BIN

acieroid committed:
[deployment] Try to set CHROME_BIN

acieroid committed:
[deployment] Really trying to get chromedriver to work

acieroid committed:
[deployment] Really trying to get chromedriver to work

acieroid committed:
[deployment] Disable image generation to gain build time

acieroid committed:
[deployment] Disable image generation to gain build time

acieroid committed:
[deployment] Simply build script as much as possible to reduce build time

acieroid committed:
[deployment] Simply build script as much as possible to reduce build time

acieroid committed:
[deployment] Update CI workflows

acieroid committed:
[deployment] Update CI workflows

acieroid committed:
[deployment] Use newly defined action in workflow

acieroid committed:
[deployment] Use newly defined action in workflow

acieroid committed:
[deployment] Syntax error in action yaml

acieroid committed:
[deployment] Syntax error in action yaml

acieroid committed:
[deployment] Perform checkout before using custom action

acieroid committed:
[deployment] Perform checkout before using custom action

acieroid committed:
[deployment] Try to get custom action working

acieroid committed:
[deployment] Try to get custom action working

acieroid committed:
[deployment] Move custom action to subdirectory

acieroid committed:
[deployment] Move custom action to subdirectory

acieroid committed:
[deployment] Fix custom action

acieroid committed:
[deployment] Fix custom action

acieroid committed:
[deployment] Further simplify custom action

acieroid committed:
[deployment] Further simplify custom action

acieroid committed:
[deployment] Uses inputs for custom action

acieroid committed:
[deployment] Uses inputs for custom action

acieroid committed:
[deployment] Fix typo

acieroid committed:
[deployment] Fix typo

acieroid committed:
[deployment] Complete overhaul of CI scripts

acieroid committed:
[deployment] Complete overhaul of CI scripts

acieroid committed:
[deployment] Update scripts to get full CI pipeline working hopefully

acieroid committed:
[deployment] Update scripts to get full CI pipeline working hopefully

acieroid committed:
[deployment] revers eslint version to 7 for compatibility with old angular

acieroid committed:
[deployment] revers eslint version to 7 for compatibility with old angular

acieroid committed:
[deployment] Update dummy firebaseConfig to please linter

acieroid committed:
[deployment] Update dummy firebaseConfig to please linter

acieroid committed:
[deployment] Add maintenance mode easily deployable

acieroid committed:
[deployment] Add maintenance mode easily deployable

acieroid committed:
[deployment] Test netlify deployment trigger

acieroid committed:
[deployment] Test netlify deployment trigger

acieroid committed:
[deployment] Trigger build through webhook

acieroid committed:
[deployment] Trigger build through webhook

acieroid committed:
[deployment] Use netlify environment

acieroid committed:
[deployment] Use netlify environment

acieroid committed:
[deployment] Try to use netlify CLI to trigger deployment

acieroid committed:
[deployment] Try to use netlify CLI to trigger deployment

acieroid committed:
[deployment] Forgot secrets

acieroid committed:
[deployment] Forgot secrets

acieroid committed:
[deployment] More experiments with deployment, now from github

acieroid committed:
[deployment] More experiments with deployment, now from github

acieroid committed:
[deployment] Forgot to checkout

acieroid committed:
[deployment] Forgot to checkout

acieroid committed:
[deployment] Use GH secrets

acieroid committed:
[deployment] Use GH secrets

acieroid committed:
Merge branch 'develop' of github.com:MartinREMY42/multi-game-project into deployment

acieroid committed:
[deployment] Fix syntax

acieroid committed:
[deployment] Fix syntax

acieroid committed:
[deployment] Really fix syntax issues

acieroid committed:
[deployment] Really fix syntax issues

acieroid committed:
[deployment] Commit new translations

acieroid committed:
[deployment] CD: check if user is member of the organization

acieroid committed:
[deployment] CD: check if user is member of the organization

acieroid committed:
[deployment] Improve deployment on master/develop/feature branch

acieroid committed:
[deployment] Improve deployment on master/develop/feature branch

acieroid committed:
[deployment] Try to pass username to action

acieroid committed:
[deployment] Try to pass username to action

acieroid committed:
[deployment] Trying with quotes

acieroid committed:
[deployment] Trying with quotes

acieroid committed:
[deployment] Update action version

acieroid committed:
[deployment] Update action version

acieroid committed:
[deployment] Fix action output usage

acieroid committed:
[deployment] Fix action output usage

acieroid committed:
[deployment] Try fix for org member

acieroid committed:
[deployment] Try fix for org member

acieroid committed:
[deployment] Try to get the branch name from the PR

acieroid committed:
[deployment] Try to get the branch name from the PR

acieroid committed:
[deployment] Add some debugging info

acieroid committed:
[deployment] Add some debugging info

acieroid committed:
[deployment] Try to get branch name in CI

acieroid committed:
[deployment] Try to get branch name in CI

acieroid committed:
Merge branch 'develop' of github.com:MartinREMY42/multi-game-project into deployment

acieroid committed:
[deployment] Clean deployment script

acieroid committed:
[deployment] Clean deployment script

acieroid committed:
[deployment] Update python dependencies

acieroid committed:
[deployment] Update python dependencies

acieroid committed:
[deployment] Update python deps, once more

acieroid committed:
[deployment] Update python deps, once more

acieroid committed:
[deployment] Trigger new deployment

acieroid committed:
Merge branch 'develop' of github.com:MartinREMY42/multi-game-project into deployment

acieroid committed:
[deployment] Update package-lock.json after merge

acieroid committed:
[deployment] Update package-lock.json after merge

acieroid committed:
[deployment] Update translations, update image script

acieroid committed:
[deployment] Update translations, update image script

acieroid committed:
[deployment] Experiment with CD: try to get branch deploy with prod

acieroid committed:
[deployment] Experiment with CD: try to get branch deploy with prod

acieroid committed:
[deployment] Try netlify build

acieroid committed:
[deployment] Try netlify build

acieroid committed:
[deployment] Add build script for netlify

acieroid committed:
[deployment] Add build script for netlify

acieroid committed:
Revert "[deployment] Add build script for netlify"

This reverts commit 3688b2e9f61037896170d263fa06cfd422981640.

acieroid committed:
Revert "[deployment] Add build script for netlify"

This reverts commit 3688b2e9f61037896170d263fa06cfd422981640.

acieroid committed:
Revert "[deployment] Try netlify build"

This reverts commit a53bbc9aed0bc622dc96075c07f0aa13b5817dfb.

acieroid committed:
Revert "[deployment] Try netlify build"

This reverts commit a53bbc9aed0bc622dc96075c07f0aa13b5817dfb.

acieroid committed:
Revert "[deployment] Experiment with CD: try to get branch deploy with prod"

This reverts commit 5f9309d5415814c3ae184ac22ebd1d949b22568d.

acieroid committed:
Revert "[deployment] Experiment with CD: try to get branch deploy with prod"

This reverts commit 5f9309d5415814c3ae184ac22ebd1d949b22568d.

acieroid committed:
Merge pull request #1 from EveryBoard/deployment

Deployment

acieroid committed:
[deployment] Update netlify-cli dependency

acieroid committed:
[deployment] Update netlify-cli dependency

acieroid committed:
[deployment] Add build script back

acieroid committed:
[deployment] Add build script back

acieroid committed:
[deployment] Trigger CI build

acieroid committed:
[deployment] Use github pages for master/dev

acieroid committed:
[deployment] Use github pages for master/dev

acieroid committed:
Merge pull request #2 from EveryBoard/deployment

Deployment

acieroid committed:
[deployment] Fix dependencies for gh-pages deployment

acieroid committed:
[deployment] Fix dependencies for gh-pages deployment

acieroid committed:
[deployment] Enable logging to see what went wrong

acieroid committed:
[deployment] Enable logging to see what went wrong

acieroid committed:
[deployment] Add email info in ng deploy

(Lunch break commit)

acieroid committed:
[deployment] Add email info in ng deploy

(Lunch break commit)

MartinREMY42 committed:
[Brandhub] Make tafl component adapted to different sized boards

acieroid commented on scripts/games.txt:
```diff
@@ -0,0 +1,25 @@
+MinimaxTesting
```
t'as lancé le script de screenshot qui a planté ? Car ce fichier devrait avoir été supprimé !

acieroid committed:
[deployment] Finalize new workflow

acieroid committed:
[deployment] Finalize new workflow

acieroid committed:
[deployment] Rename pantheonsgame to everyboard

acieroid committed:
[deployment] Rename pantheonsgame to everyboard

acieroid committed:
[conspirateurs] Implement moves and state, draft rules

acieroid committed:
Merge branch 'master' of github.com:MartinREMY42/multi-game-project into develop-merged

acieroid committed:
Merge branch 'develop' of github.com:EveryBoard/EveryBoard into develop-merged

acieroid committed:
Update package-lock

acieroid committed:
[conspirateurs] Finalize rules

acieroid committed:
[conspirateurs] First draft of component

acieroid committed:
Get up to date with the latest develop

acieroid committed:
Merge branch 'develop' of github.com:MartinREMY42/multi-game-project into conspirateurs

acieroid committed:
[conspirateurs] Start testing component

acieroid committed:
Merge pull request #3 from EveryBoard/develop-merged

Develop merged

acieroid committed:
Merge develop from upstream

acieroid committed:
[conspirateurs] Start working on minimax

acieroid committed:
Update angular.json

acieroid committed:
Update dependencies

acieroid committed:
Finalize the name change

acieroid committed:
Finalize the name change, really

acieroid committed:
[conspirateurs] Add minimax spec

acieroid committed:
Merge branch 'develop' of github.com:EveryBoard/EveryBoard into conspirateurs

acieroid committed:
Update translations

acieroid committed:
Merge branch 'develop' of github.com:EveryBoard/EveryBoard into conspirateurs

acieroid committed:
Fix root path in new environment

acieroid committed:
[conspirateurs] Some more progress, and improve coverage of Minimax.ts

acieroid committed:
[conspirateurs] Finish tutorial

acieroid committed:
[conspirateurs] Get full coverage

acieroid committed:
[conspirateurs] Add images

acieroid committed:
[conspirateurs] Fix broken test and remove console.logs

acieroid committed:
[part-creation-coverage] Refactor part creation tests

  - add GWT
  - improve tests structure
  - get 100% coverage
  - did not encounter heisentest

acieroid committed:
[part-creation-coverage] Fix linter issues, restore original HTML

acieroid committed:
[conspirateurs] PR comments

acieroid committed:
[conspirateurs] More PR comments

acieroid committed:
[conspirateurs] Test getBoardValue scores

acieroid committed:
[conspirateurs] Improve jump visualisation and minimax

acieroid committed:
[conspirateurs] Improve minimax, i18n

acieroid committed:
[part-creation-coverage] PR comments

acieroid committed:
[conspirateurs] PR comments, wave 2

acieroid committed:
[conspirateurs] Enable jump cancellation upon other piece click

acieroid committed:
[conspirateurs] Improve shelter highlights

MartinREMY42 committed:
Merge pull request #5 from EveryBoard/part-creation-coverage

Refactor part creation tests

acieroid committed:
[conspirateurs] PR comments

acieroid committed:
Merge branch 'develop' of github.com:EveryBoard/EveryBoard into HEAD

acieroid committed:
[conspirateurs] Change victory visualization, fix some previously useless tests

acieroid committed:
[conspirateurs] Fix linter and update translations

acieroid committed:
[conspirateurs] PR comment + update game images

MartinREMY42 committed:
Merge pull request #4 from EveryBoard/conspirateurs

New game: conspirateurs

MartinREMY42 committed:
[Brandhub] removed games.txt temporary file

MartinREMY42 committed:
Merge branch 'develop' of https://github.com/EveryBoard/EveryBoard into Brandhub
